Sources/AsyncHTTPClient/ConnectionPool.swift

```diff
@@ -234,6 +240,13 @@ class HTTP1ConnectionProvider {
         self.closePromise = eventLoop.makePromise()
         self.state = .init(eventLoop: eventLoop)
         self.backgroundActivityLogger = backgroundActivityLogger
+
+        self.factory = HTTPConnectionPool.ConnectionFactory(
+            key: self.key,
+            tlsConfiguration: tlsConfiguration ?? configuration.tlsConfiguration ?? .forClient(),
```

Comment:
Minor cleanup, purely personal opinion, I'd refactor out this line into a `let`

Reply:
This is totally unneeded here. Removed :)